style_guide.md

```diff
@@ -140,9 +142,26 @@ you will -> you'll
 
 ### Abbreviations
 
-Spell out abbreviations the first time they are used.
+Abbreviations include acronyms, initialisms, shortened words, and contractions.
 
-Ex: United States (U.S.)
+- Spell out the term the first time it's used, followed by the abbreviation in parentheses. Example: Mobile Security Testing Guide (MSTG). Subsequent usages in the same chapter may include the abbreviation only.
+- If it only appears once in the content, spell out the term instead of using the abbreviation.
+- In titles and headings, use the abbreviation but be sure to properly introduce it (see above) in the text that follows.
+- Use "a" or "an" depending on the pronunciation (as with regular words). Example: a DLL, an APK, a URL.
```

Comment:
```suggestion
- Use "a" or "an" depending on the pronunciation of the acronym. Example: a DLL, an APK, a URL, a SQL.
```

I don't remember which one, but there are a few that have a different pronunciation depending on if you say the abbreviation or the full words. Let's make the acronym the default one.

I also added the SQL example as it should be pronounced "sequel", but many people pronounce it as es-que-el.